README.md

```diff
@@ -130,6 +134,29 @@ To reach the project support team, email [pyansys.core@ansys.com](mailto:pyansys
 To run PyAEDT, you must have a local licenced copy of AEDT.
 PyAEDT supports AEDT versions 2022 R1 or newer.
 
+## Contributing
+
+See the [PyAnsys develoeprs guide](https://dev.docs.pyansys.com) for detailed information about contributing to the PyAnsys project. 
+
+Note that PyAEDT uses semantic naming for pull requests (PR). This convention greatly simplifies
+
+the review process by providing meaningful information in the PR title. The

```

Comment:
```suggestion
the review process by providing meaningful information in the PR title. The

```
Remove extra line? Wouldn't you be better off linking to the dev guide topic for these prefixes rather than having to maintain a list here?

Reply:
@PipKat  I couldn't find the equivalent section in the developers guide, but will add a link to ansys/actions/commit-style. This list comes directly from that location.